### PR TITLE
Port change for phoenix server

### PR DESCRIPTION
**Pull request by force10269**
Changed localhost port from 5000 to 3000

### CODE REVIEW

File: server-phoenix/priv/static/manifest.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  "media/images/commit.png": "/media/images/commit-2c662617.png",
+  "media/images/displayCommit.png": "/media/images/displayCommit-8a84c3c9.png",
+  "media/images/displayPR.png": "/media/images/displayPR-ca202dda.png",
+  "media/images/displaySlack.png": "/media/images/displaySlack-62d96504.png",
+  "media/images/logo.png": "/media/images/logo-612932ca.png",
+  "media/images/pr.png": "/media/images/pr-e808acd6.png",
+  "media/images/raincloud.png": "/media/images/raincloud-f4e49358.png",
+  "media/images/slack.png": "/media/images/slack-1765e252.png",
+  "media/images/sunmoon.png": "/media/images/sunmoon-0ecf3389.png"
+}
```
Comment (cbarber) [inline server-phoenix/priv/static/manifest.json:43]: Looks like this got committed by mistake. My mistake for not adding it to `.gitignore`

**Review by cbarber — COMMENTED** [review]
It looks like there may be a couple other places to update: `git grep ":5000"`

You can ignore `*.rb` files since rails is no longer used.

**Review by cbarber — APPROVED** [review]
Nice atomic commits :100:

File: .gitignore [hunk 2]
```diff
@@ -52,7 +52,9 @@ yarn-debug.log*
 
 # Ignore manifest
 /public/manifest.json
+/server-phoenix/priv/static/manifest.json
 
 # Ignore newly generated sqlite3 db
 /db/development.sqlite3-shm
 /db/development.sqlite3-wal
+/db/
```
Comment (allisonlucas) [inline .gitignore:60]: What is this addition doing?

Comment (force10269) [reply .gitignore:60]: I did this with Cathy, she talked about how it came from SourceAdvisors and that it also comes with the standard .gitignore generated by Phoenix

**Pull request merged**